.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: c
+os:
+  - linux
+script:
```

Comment:
this can easily grow. rather than do this, how about we create a test/travis.sh script to handle this logic and this line simply reads:

```
script: ./test/travis.sh
```

Reply:
Done, please review.